ditto/metrics/network_analysis.py

```diff
@@ -502,6 +585,11 @@ def analyze_object(self, obj, feeder_name):
                     else:
                         self.points[feeder_name]=[[X,Y]]
```

Comment:
Why is this stored as a list of the list `[X, Y]`?

Reply:
I store all the coordinates in a 2D array: 

```
[  [X_node1, Y_node1],
   [X_node2, Y_node2],
...
]
```

Such that I can compute the convex hull of the feeders.